guake/boxes.py

```diff
@@ -381,9 +381,8 @@ def set_search(self, widget):
 
             # Set search regex on term
             self.searchstring = text
-            self.searchre = GLib.Regex(text, 0, 0)
-            term.search_set_gregex(self.searchre, 0)
-
+            self.searchre = Vte.Regex.new_for_search(text, -1, Vte.REGEX_FLAGS_DEFAULT)
```

Comment:
As [vte doc note](https://valadoc.org/vte-2.91/Vte.Terminal.search_set_regex.html), we should create regex  with `PCRE2_MULTILINE` flag.

Please use `Vte.REGEX_FLAGS_DEFALUT | PCRE2_MULTILINE`, please also add PCRE2_MULTILINE to `globals.py`, similar to https://github.com/gnome-terminator/terminator/blob/6c56f32eeb4306371842bc1aba17b1b677ec0352/terminatorlib/regex.py#L13

Reply:
```
(guake:58545): VTE-WARNING **: 13:24:24.629: (../vte/src/vtegtk.cc:2905):void vte_terminal_search_set_regex(VteTerminal*, VteRegex*, guint32): runtime check failed: (regex == nullptr || _vte_regex_has_multiline_compile_flag(regex))

(guake:58545): VTE-WARNING **: 13:24:24.806: (../vte/src/vtegtk.cc:2905):void vte_terminal_search_set_regex(VteTerminal*, VteRegex*, guint32): runtime check failed: (regex == nullptr || _vte_regex_has_multiline_compile_flag(regex))
```